mcp-servers/helm/templates/scc-bindings.yaml

```diff
@@ -0,0 +1,59 @@
+{{/*
+Always create SCC binding for toolhive-operator since it's required even when operator is installed externally
+*/}}
+{{- if true }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Release.Name }}-toolhive-operator-anyuid
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+subjects:
+- kind: ServiceAccount
+  name: toolhive-operator
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: system:openshift:scc:anyuid
+  apiGroup: rbac.authorization.k8s.io
+{{- end }}
+
+{{- if index .Values "mcp-servers" "mcp-weather" "mcpserver" "enabled" }}
```

Comment:
@rhkp we should probably change it to a list so we can loop over the objects and check if .enabled